pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,202 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). For fine control over long–lived agents, you might like to run the agent using individual VM instances on Google Compute Engine. Or run Docker–based builds using a scalable cluster of agents on the Google Container Engine.
+
+<%= toc %>
+
+## Running the agent on Google Compute Engine
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example:
+
+Launch an instance using the latest Ubuntu LTS image [through the console](https://console.cloud.google.com/compute/instancesAdd#preconfigured-image-ubuntu-1604-xenial-v20170202):
+
+<%= image "create-an-instance.png", size: "2200x1216", alt: "Screenshot of creating a Google Compute Engine instance running Ubuntu 16.04 LTS via the Google Cloud Console" %>
+
+Connect via SSH:
+
+<%= image "ssh-to-instance.png", size: "2048x1220", alt: "Screenshot of connecting to a Google Compute Engine instance via the Google Cloud Console" %>
+
+Now follow the [setup instructions for Ubuntu](/docs/agent/ubuntu).
+
+## Running the agent on Google Container Engine
+
+[Google Container Engine](https://cloud.google.com/container-engine) can run the agent as a [Docker](https://www.docker.com) container using [Kubernetes](https://kubernetes.io). If you wish to [run Docker–based builds](/docs/builds/docker-containerized-builds), ensure the container is started with a [privileged security context](https://kubernetes.io/docs/user-guide/pods/#privileged-mode-for-pod-containers) and mount the the Docker socket and binary as volumes. For example:
+
+Start a Google Container Engine cluster [through the console](https://console.cloud.google.com/kubernetes/add):
+
+<%= image "create-a-cluster.png", size: "2048x1216", alt: "Screenshot of creating a Google Container Engine cluster via the Google Cloud Console" %>
+
+Open [Google Cloud Shell](https://cloud.google.com/shell/) or your own console with [gcloud](https://cloud.google.com/sdk/gcloud/) installed and authenticated. You'll need to configure kubectl to talk to your new cluster. The console includes a "Connect" button which shows the exact command to run:
+
+<%= image "connect-to-cluster.png", size: "2048x1216", alt: "Screenshot of cluster connection settings in the Google Cloud Console" %>
+
+Running it should look a bit like this, with your details in the right places:
```

Comment:
A bit? Or actually exactly like this (with their info)? I think that 'substituted in' might work better than 'in the right places' too.